great_expectations/rule_based_profiler/parameter_builder/metric_multi_batch_parameter_builder.py

```diff
@@ -90,9 +106,16 @@ def _build_parameters(
             variables=variables,
             parameters=parameters,
         )
+        metric_values: Union[
+            np.ndarray, List[Union[Any, Number]]
```

Comment:
From the perspective of allowing others to understand what to expect a type to be, a `Union[Any, ...]` is not much different than just saying `Any`.

I think we should be explicit about what a metric result could be

Reply:
@donaldheppner We cannot right now -- but could very formally if we did [IPV-250] (please see my reply to your previous comment).  Thanks.

Reply:
@donaldheppner Please see an earlier comment -- I made a temporary encapsulation measure using dataclasses.